lib/bootloader_setup.pm

```diff
@@ -739,6 +739,12 @@ sub specific_bootmenu_params {
         return " @params ";
     }
 
+    # Enable kernel.softlockup_panic, unless explicitly disabled
+    # See bsc#1126782
+    if (!get_var("SOFTLOCKUP_PANIC_DISABLED", 0)) {
+        push @params, "kernel.softlockup_panic=1";
+    }
```

Comment:
maybe better

```perl
push @params, 'kernel.softlockup_panic=1' unless get_var('SOFTLOCKUP_PANIC_DISABLED', 0);
```

so single ticks where you don't want to evaluate variables and use the common post-condition style, especially with "unless", with the benefit of saving lines for readability.

Reply:
I actually can't remember why I didn't do it like that back then